taxi/urls.py

```diff
@@ -1,5 +1,9 @@
+from mailcap import lookup
+
+from django.contrib.auth import logout
```

Comment:
The import `from django.contrib.auth import logout` is not used in this file and can be removed to clean up the code.